asapdiscovery-data/asapdiscovery/data/schema.py

```diff
@@ -1,7 +1,7 @@
-from typing import Dict, List
```

Comment:
@mikemhenry is there a reason why this was changed? Now I get this error on import:
```
Traceback (most recent call last):
  File "/Users/alexpayne/Scientific_Projects/covid-moonshot-ml/asapdiscovery-modeling/asapdiscovery/modeling/tests/prep_mers_rcsb/test-cif1-to-prepped-du.py", line 30, in <module>
    from asapdiscovery.docking.modeling import align_receptor
  File "/Users/alexpayne/Scientific_Projects/covid-moonshot-ml/asapdiscovery-docking/asapdiscovery/docking/__init__.py", line 10, in <module>
    from .modeling import *  # noqa: F403,F401
  File "/Users/alexpayne/Scientific_Projects/covid-moonshot-ml/asapdiscovery-docking/asapdiscovery/docking/modeling.py", line 15, in <module>
    from asapdiscovery.data.schema import CrystalCompoundData
  File "/Users/alexpayne/Scientific_Projects/covid-moonshot-ml/asapdiscovery-data/asapdiscovery/data/schema.py", line 11, in <module>
    class ExperimentalCompoundData(Model):
  File "/Users/alexpayne/Scientific_Projects/covid-moonshot-ml/asapdiscovery-data/asapdiscovery/data/schema.py", line 43, in ExperimentalCompoundData
    experimental_data: dict[str, float] = Field(
TypeError: 'type' object is not subscriptable
```

I think this is because it it using the default `dict` type instead of the typing.Dict object.

Reply:
This was addressed in https://github.com/choderalab/asapdiscovery/pull/157 we were masking the `list` built-in keyword

Reply:
Oh, this probably means that there's something masking the `dict` built-in keyword. Or you are using a python version that's not supported.

Reply:
I think you need python 3.9+ for this not to be a problem. I guess we decided not to support older python versions.

Reply:
if I revert this change as in this commit:
https://github.com/choderalab/asapdiscovery/pull/104/commits/bd351de95d696e9f70083682b50fc45ac805fc4d

The error goes away. The schema being used by @kaminow here uses special types (?). But we aren't overwriting the defaults, we're just using a different object (Dict vs dict)

Reply:
OH i see

Reply:
I'm using python 3.8, I can try to upgrade to 3.9 and see if anything breaks